app.js

```diff
@@ -6,11 +6,25 @@ const express = require('express')
 //Node.JS package that helps manage paths
 const path = require("path")
 
+//using node.js's require function to import the body-parser
+//npm module
+const bodyParser = require("body-parser")
+
+//using node.js's require function to import the index routing file
+const index = require('./routes/index')
+
 //calls the express function "express()" and puts new Express
 //application inside the app variable. "express()" is a class and app is it's
 //newly created object
 const app = express()
 
+//mounts the middleware function (body-parser's urlendcoded method)
+//since the path defaults to '/', middleware mounted without a path will be
+//executed for every request to the app
+app.use(bodyParser.urlencoded({
+    extended: false
+}))
+
 //Routes HTTP GET requests to the uri with the specified
 //callback function.
```

Comment:
When an HTTP GET request is sent to the path, run the callback function.

This is the coach of the ping pong player on the server side telling the server player to listen for this signal and telling them what to do when that happens.